pkg/amphoracontrollers/volumes.go

```diff
@@ -0,0 +1,55 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package amphoracontrollers
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	configVolume = "amphora-certs"
+)
+
+var (
+	// Files get mounted as root:root, but process is running as octavia
+	configMode int32 = 0644
```

Comment:
I think this is okay for now but we should look at if its possible to mount as the octavia user.